ReSharper.FSharp/test/data/parsing/arguments/Tuple param 03 - Param not deconstructed.gold

```diff
@@ -0,0 +1,11 @@
+﻿module M =
+    let f a |z|(param #2) =
+        let x, y = a
+        x + y + z
+
+M.f (|1|(arg #0), |2|(arg #1)) |3|(arg #2)
+
+---------------------------------------------------------
+(arg #0) => <no matching param>
+(arg #1) => <no matching param>
```

Comment:
I wonder if we could somehow make parameter groups for curried functions and match`a` here later.